Content.Server/GameObjects/Components/NodeContainer/Nodes/Node.cs

```diff
@@ -127,5 +149,22 @@ private INodeGroup MakeNewGroup()
         {
             return _nodeGroupFactory.MakeNodeGroup(NodeGroupID);
         }
+
+        private void AnchorUpdate()
+        {
+            if (Owner.GetComponent<PhysicsComponent>().Anchored)
```

Comment:
Why not do `if (Connectable)` or `if (Anchored)`? I appreciate that this *should* only be called when a `PhysicsComponent` exists, but the properties would improve readability.

Reply:
Replaced with if (Anchored)